### PR TITLE
[Feature1] Feature1 changes are to be implemented

### PR DESCRIPTION
**Pull request by vivekkulkarni-ops**
Blah blah

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,6 @@
-# first-github-repo
\ No newline at end of file
+# first-github-repo
```
Comment (kulkvivek) [inline README.md:1]: This is pretty good.
```suggestion
# first-github-repo
```

File: README.md [hunk 2]
```diff
@@ -1 +1,6 @@
...
+Bakchodi patakni hai
```
Comment (kulkvivek) [inline README.md:2]: Nothing here. Probably could work more on the dynamic programming,

File: README.md (hunk 1 above)
Comment (vivekkulkarni-ops) [reply README.md:1]: Thank you very mych

File: gen1.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+Temperature 89k
```
Comment (vivekkulkarni-ops) [inline gen1.txt:2]: Use degree celciyus

Comment (kulkvivek) [inline gen1.txt:2]: Don't use degree celcius. Use fahrenheit

**Review by kulkvivek — CHANGES_REQUESTED** [review]

File: gen1.txt (hunk 3 above)
Comment (vivekkulkarni-ops) [reply gen1.txt:2]: > Don't use degree celcius. Use fahrenheit

Okau

Comment (kulkvivek) [inline gen1.txt:2]: No

**Review by kulkvivek — APPROVED** [review]

**Pull request merged**